index.js

```diff
@@ -4,15 +4,42 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "blue";
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector("#profile")
+console.log(profile)
+const travelerImg = profile.querySelector("img")
+const travelerName = profile.querySelector("h2")
+const travelerNickname = profile.querySelector("em")
+travelerImg.src = traveler.photo;
+travelerImg.alt = "Profile Pic"
+travelerName.textContent = traveler.name;
+travelerNickname.textContent = traveler.nickname;
 
```

Comment:
Yes!! I don't see `traveler` initialized anywhere, however though after you have the `traveler` object you can then do `traveler.photo`, `traveler.name`, etc.